db/schema.sql

```diff
@@ -0,0 +1,36 @@
+DROP DATABASE IF EXISTS trailhike_db;
+CREATE database trailhike_db;
+
+USE trailhike_db;
+
+CREATE TABLE user (
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(45) NOT NULL,
+    email VARCHAR(45) NOT NULL,
+    password VARCHAR(45) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE trail (
```

Comment:
@polodre Since we're using Sequelize to define Models, that define our database tables, there isn't a need to also create the tables with plain SQL in the schema file. I would suggest removing the create tables commands from the schema file. If they stay and in the future we update the models we would also need to update the create table command, otherwise it might cause a conflict. Best to define the model/table in just one place so easier to modify.